v2/internal/frontend/runtime/desktop/events.js

```diff
@@ -107,7 +113,11 @@ function notifyListeners(eventData) {
         }
 
         // Update callbacks with new list of listeners
-        eventListeners[eventName] = newEventListenerList;
+        if (newEventListenerList.length === 0) {
+            removeListener(eventName);
```

Comment:
@leaanthony hey, note this line, I had to add this so WailsInvoke got called with `EX${eventName}`, which I think was the intent (as I captured here https://github.com/wailsapp/wails/pull/1969/files#diff-eda323563cc3726a10525d8ae664e9cd8ba98fce7d961ed8b8d6c634670506d1R39). Does that look right to you?

Reply:
Yeah I think so 👍